fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body class="container">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+      
+      <form id="survey-form">
+        <div class="input-container">
+          <div class="input-description">
+            <label id="name-label" for="name">Name</label>
+            <input required id="name" type="text" placeholder="Enter your name">
+          </div>
+          <div class="input-description">
+            <label id="email-label" for="email">Email</label>
+            <input required id="email" pattern=".+@globex\.com" type="email" placeholder="Enter your email">
+          </div>
+          <div class="input-description">
+            <label id="number-label" for="number">Age</label>
+            <input required id="number" type="number" placeholder="Enter a number" min="10" max="100">
+          </div>          
+        </div>        
+
+        <p class="center">Which option best describes your current role?</p>
+        <div class="selectContainer">
+          <select id="dropdown" name="s">
+            <option selected disabled value="">Select current role</option>
+            <option value="">Student</option>
+            <option value="">Full time job</option>
+            <option value="">Full time lerner</option>
+            <option value="">Preder not to say</option>
```

Comment:
It's important to specify the content of the **value** attribute.
**Note:** the value **attribute** specifies the initial value of the element.

Reply:
Excellent in every way 😎

Reply:
#### OPTIONAL

This is just a suggestion so you can apply it if you wish 😁To improve your reviews even further, you could use a couple of emojis and/or gifs in the review. The reason for this is to make the code review more human-friendly 👽🤖